CIFAR10/train/ajay_resnet9_1v100_pytorch.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "v1.0",
+    "author": "Ajay Uppili Arasanipalai, Elliptigon",
+    "authorEmail": "ajay@elliptigon.com",
+    "framework": "pytorch 0.4.0",
+    "codeURL": "https://github.com/iyaja/cifar10-fast-2", 
+    "model": "Custom ResNet 9",
+    "hardware": "V100 (Nimbix np9g1)",
+    "costPerHour": 1.68,
```

Comment:
![Screenshot 2019-04-17 at 11 35 22 AM](https://user-images.githubusercontent.com/30197649/56264689-e7e7c380-6104-11e9-8412-89211cd07544.png)

I wasn't able to find a site with their instance prices, but here's a screenshot from my portal. Nimbix requires you to contact their team before getting acess to their services. I have acess through a personal contact.

Reply:
[Salamander.ai](https://salamander.ai) has similar pricing for the same instance.

Reply:
@iyaja Using the [Nimbix's price calculator](https://www.nimbix.net/cloud-price-calculator/#/), I got a cost per hour of $5.60 for an np9g1 instance (see below), so it seems like your account has some special discount. Aside from the `costPerHour` field, your submission looks okay. You can either (1) update the `costPerHour` field to $5.60 or (2) remove `costPerHour` for now so I can merge this pull request and add it back in a separate PR where we can discuss the correct cost. Please let me know how you want to proceed.

<img width="1010" alt="Screen Shot 2019-04-20 at 10 38 10 AM" src="https://user-images.githubusercontent.com/549702/56460536-16f67300-6359-11e9-9cf5-3320f4375595.png">

Reply:
@codyaustun yes I think something's fishy with the cost. I'm not aware of any discount on my account, but I was given access to it by someone else, so I can't say for sure.

At the same time, it would be unfair to list $1.68 as the cost per hour as this would not be reproducible.

I'll try getting the same cost and performance on salamandar.ai in the next few days.

Until then, please go ahead and merge the pull request. I have removed the cost field for now.